code/game/objects/items/devices/translator.dm

```diff
@@ -4,7 +4,7 @@
 	desc = "This handy device appears to translate the languages it hears into onscreen text for a user."
 	icon = 'icons/obj/device.dmi'
 	icon_state = "translator"
-	w_class = ITEMSIZE_SMALL
+	w_class = ITEMSIZE_NORMAL //VOREStation Edit
```

Comment:
This makes it not fit in pockets. That's the only thing I think is bad. Fitting in pockets is totally reasonable. It's _handheld_ after all.

Reply:
That was mainly to stop people turning it on and putting it in their pocket and still being able to use it, since it's supposed to read the translation out on a screen (which doesn't make sense in a pocket).
People used to use voice recorders as translators, and that required using them in-hand to get the translation.

Reply:
Fair enough.